GitDemo/Program.cs

```diff
@@ -7,7 +7,18 @@ namespace GitDemo
     internal class Program
     {
         private static void Main(string[] args)
-            => Console.WriteLine(GetMessage());
+        {
+            Console.WriteLine("Headline");
+            Console.WriteLine("===========");
+            Console.WriteLine(GetMessage());
+            Console.Write("Checksum word: ");
+            var tag = Console.ReadLine();
+            var someThing = new SomeThing(tag);
+            Console.Write("The checksum is: ");
+            someThing.OnNext();
+            Console.WriteLine(someThing.Result);
+            Console.ReadLine();
+        }
 
         internal static string GetMessage()
             => "Hello Airsphere Team!";
```

Comment:
nENENE SO GEHT DAS GAR NICHYT!!!!
```suggestion
            => "Hello AirSphere Team!";
```